assets/pages/bundles-landing/helpers/blackFriday.js

```diff
@@ -45,8 +47,40 @@ function getPaperItems() {
   return [items[0], { heading: 'Save money on the retail price' }, items[1]];
 }
 
+function getPaperItemsForStructureTest() {
+  const items = [
+    {
+      heading: 'Choose your package and delivery method',
+      text: 'Everyday, Sixday, Weekend and Sunday; redeem paper vouchers or get home delivery',
+    },
+  ];
+
+  if (inOfferPeriod()) { return [offerItem, ...items]; }
+
+  return [items[0], { heading: 'Save money on the retail price' }];
+}
+
+function getPaperDigitalItemsForStructureTest() {
+
+  const items = [
+    {
+      heading: 'Choose your package and delivery method',
+      text: 'Everyday, Sixday, Weekend and Sunday; redeem paper vouchers or get home delivery',
+    },
+    {
+      heading: 'Get all the benefits of a digital subscription',
+    },
+  ];
+
+  if (inOfferPeriod()) { return [offerItem, ...items]; }
+
+  return [items[0], { heading: 'Save money on the retail price' }, items[1]];
```

Comment:
move `{ heading: 'Save money on the retail price' }` into a top level const and use it here and in getPaperItems()